websites/H/Hulu/presence.ts

```diff
@@ -8,218 +8,261 @@ const strings = presence.getStrings({
-      '.SimpleModalNav__brandImage',
-    )
-    item = document.querySelector('.Subnav__item.active')
-    details = 'Viewing Network'
-    if (brand) {
-      state = brand.alt
-      if (item)
-        state = `${state}'s ${item.textContent}`
-    }
-  }
-  else if (path.includes('/sports_episode')) {
-    title = document.querySelector('.Masthead__title')
-    item = document.querySelector('.Subnav__item.active')
-    details = 'Viewing Sports Episode'
-    if (title) {
-      state = title.textContent
-      if (item)
-        state = `${state}'s ${item.textContent}`
-    }
-  }
-  else if (path.includes('/sports_team')) {
-    title = document.querySelector('.Masthead__title')
-    item = document.querySelector('.Subnav__item.active')
-    details = 'Viewing Sports Team'
-    if (title) {
-      state = title.textContent
-      if (item)
-        state = `${state}'s ${item.textContent}`
+  switch (true) {
+    case path.includes('/hub'):
+      header = document.querySelector('.Hub__title')
+      title = document.querySelector('.SimpleModalNav__title')
+      presenceData.details = (await strings).viewCategory
+      if (header) {
+        presenceData.state = header.textContent
+        if (title)
+          presenceData.state = `${presenceData.state} (${title.textContent})`
+      }
+      break
+    case path.includes('/genre'):
+      header = document.querySelector('.Hub__title')
+      title = document.querySelector('.SimpleModalNav__title')
+      presenceData.details = (await strings).viewGenre
+      if (header) {
+        presenceData.state = header.textContent
+        if (title)
+          presenceData.state = `${presenceData.state} (${title.textContent})`
+      }
+      break
+    case path.includes('/series'):
+      title = document.querySelector('.Masthead__title')
+      item = document.querySelector('.Subnav__item.active')
+      presenceData.details = (await strings).viewSeries
+      if (title) {
+        presenceData.state = title.textContent
+        if (item)
+          presenceData.state = `${presenceData.state}'s ${item.textContent}`
+      }
+      break
+    case path.includes('/movie'):
+      title = document.querySelector('.Masthead__title')
+      item = document.querySelector('.Subnav__item.active')
+      presenceData.details = (await strings).viewMovie
+      if (title) {
+        presenceData.state = title.textContent
+        if (item)
+          presenceData.state = `${presenceData.state}'s ${item.textContent}`
+      }
+      break
+    case path.includes('/network'): {
+      const brand = document.querySelector<HTMLImageElement>(
+        '.SimpleModalNav__brandImage',
+      )
+      item = document.querySelector('.Subnav__item.active')
+      presenceData.details = (await strings).viewNetwork
+      if (brand) {
+        presenceData.state = brand.alt
+        if (item)
+          presenceData.state = `${presenceData.state}'s ${item.textContent}`
+      }
```

Comment:
Why not do it like this?
```suggestion
      if (brand && item) presenceData.state =`${brand.alt}'s ${item.textContent}`
       else if (brand && !item) presenceData.state = brand.alt    
```